compose/docker-compose-emqx.yml

```diff
@@ -65,7 +65,7 @@ services:
       - dnsconfig:/root/dnsconfig
   mq:
     container_name: mq
-    image: emqx/emqx:5.0.17
+    image: emqx/emqx:5.0.09
```

Comment:
when i try to docker-compose up with this version I get this.

Pulling mq (emqx/emqx:5.0.09)...
ERROR: manifest for emqx/emqx:5.0.09 not found: manifest unknown: manifest unknown

Reply:
that works now